cmd/awscollector/main.go

```diff
@@ -17,6 +17,7 @@ package main // import "aws-observability.io/collector/cmd/awscollector"
 
 import (
 	"fmt"
+	"github.com/aws-observability/aws-otel-collector/pkg/defaultcomponents"
```

Comment:
Is there a reason goland would automatically move this import in this order? This was done automatically and not by me.